### PR TITLE
[fix issue 68] Add listener survey.

### PR DESCRIPTION
**Pull request by jpetto**
- Also fixes small header UI bug.

### CODE REVIEW

File: layouts/partials/survey.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="survey-banner" id="survey-banner">
+    <div class="content">
+        <button type="button" id="survey-dismiss"><span>Close</span></button>
+        <p>
+            Love IRL? Take this 5 minute survey to help us make it even better.
```
Comment (stephaniehobson) [inline layouts/partials/survey.html]: This is totally a copy thing and outside my code review perview...

I'd suggest changing it to "Love IRL? You should take this..." It i[ncreases click through rates](https://kottke.org/09/07/you-should-follow-me-on-twitter).

**Comment by jpetto** [comment]
Pushed a small change to use the latest Hugo in `docker-compose.yml`, and increased the animation duration when showing the banner.

File: layouts/partials/survey.html (hunk 1 above)
Comment (thejeffreytaylor) [reply layouts/partials/survey.html]: I'm good with this.

**Comment by stephaniehobson** [comment]
This is not related to the code... I recommend removing the progress bar from the survey; it implies that there is a second page. Also labeling the button as "Done" instead of "Submit".

Maybe you can pass that on to the right person.

**Comment by jpetto** [comment]
> This is not related to the code... I recommend removing the progress bar from the survey; it implies that there is a second page. Also labeling the button as "Done" instead of "Submit".
> 
> Maybe you can pass that on to the right person.

@thejeffreytaylor What do you think?

File: src/scss/survey.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    &.visible {
+        display: block;
+    }
+}
+
+@supports (transform: translateY(100px)) {
+    .survey-banner {
+        display: block;
+        transform: translateY(200px);
+        transition: transform 0.6s cubic-bezier(0.175, 0.885, 0.320, 1.275);
```
Comment (stephaniehobson) [inline src/scss/survey.scss]: This animation is awesome 🌈

**Comment by thejeffreytaylor** [comment]
Confirming that I liked Stephanie's input and said we should go with it.

**Comment by bensternthal** [comment]
Closes https://github.com/mozilla/bedrock/issues/5674

**Pull request merged**